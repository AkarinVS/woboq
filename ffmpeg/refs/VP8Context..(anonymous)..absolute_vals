<dec f='ffmpeg/libavcodec/vp8.h' l='180' type='uint8_t'/>
<use f='ffmpeg/libavcodec/vp8.c' l='272' u='w' c='parse_segment_info'/>
<use f='ffmpeg/libavcodec/vp8.c' l='375' u='r' c='vp8_get_quants'/>
<use f='ffmpeg/libavcodec/vp8.c' l='2131' u='r' c='filter_level_for_mb'/>
<offset>8</offset>
