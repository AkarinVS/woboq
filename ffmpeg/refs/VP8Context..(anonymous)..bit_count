<dec f='ffmpeg/libavcodec/vp8.h' l='259' type='int'/>
<use f='ffmpeg/libavcodec/vp8.c' l='844' u='w' c='vp8_decode_frame_header'/>
<offset>128</offset>
