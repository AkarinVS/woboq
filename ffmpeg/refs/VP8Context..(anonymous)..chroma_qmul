<dec f='ffmpeg/libavcodec/vp8.h' l='207' type='int16_t [2]'/>
<use f='ffmpeg/libavcodec/vp8.c' l='356' u='w' c='vp7_get_quants'/>
<use f='ffmpeg/libavcodec/vp8.c' l='357' u='w' c='vp7_get_quants'/>
<use f='ffmpeg/libavcodec/vp8.c' l='385' u='w' c='vp8_get_quants'/>
<use f='ffmpeg/libavcodec/vp8.c' l='386' u='w' c='vp8_get_quants'/>
<use f='ffmpeg/libavcodec/vp8.c' l='389' u='w' c='vp8_get_quants'/>
<use f='ffmpeg/libavcodec/vp8.c' l='389' u='r' c='vp8_get_quants'/>
<use f='ffmpeg/libavcodec/vp8.c' l='389' u='r' c='vp8_get_quants'/>
<use f='ffmpeg/libavcodec/vp8.c' l='1530' u='r' c='decode_mb_coeffs'/>
<offset>64</offset>
