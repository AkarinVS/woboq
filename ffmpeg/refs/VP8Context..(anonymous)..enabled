<dec f='ffmpeg/libavcodec/vp8.h' l='179' type='uint8_t'/>
<dec f='ffmpeg/libavcodec/vp8.h' l='221' type='uint8_t'/>
<use f='ffmpeg/libavcodec/vp8.c' l='373' u='r' c='vp8_get_quants'/>
<use f='ffmpeg/libavcodec/vp8.c' l='627' u='w' c='vp7_decode_frame_header'/>
<use f='ffmpeg/libavcodec/vp8.c' l='629' u='w' c='vp7_decode_frame_header'/>
<use f='ffmpeg/libavcodec/vp8.c' l='786' u='w' c='vp8_decode_frame_header'/>
<use f='ffmpeg/libavcodec/vp8.c' l='795' u='w' c='vp8_decode_frame_header'/>
<use f='ffmpeg/libavcodec/vp8.c' l='1270' u='r' c='decode_mb_mode'/>
<use f='ffmpeg/libavcodec/vp8.c' l='2129' u='r' c='filter_level_for_mb'/>
<use f='ffmpeg/libavcodec/vp8.c' l='2136' u='r' c='filter_level_for_mb'/>
<use f='ffmpeg/libavcodec/vp8.c' l='2398' u='r' c='decode_mb_row_no_filter'/>
<use f='ffmpeg/libavcodec/vp8.c' l='2756' u='r' c='vp78_decode_frame'/>
<offset>0</offset>
<doc f='ffmpeg/libavcodec/vp8.h' l='221'>///&lt; whether each mb can have a different strength based on mode/ref</doc>
