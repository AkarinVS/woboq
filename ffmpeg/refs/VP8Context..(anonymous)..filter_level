<dec f='ffmpeg/libavcodec/vp8.h' l='184' type='int8_t [4]'/>
<use f='ffmpeg/libavcodec/vp8.c' l='278' u='w' c='parse_segment_info'/>
<use f='ffmpeg/libavcodec/vp8.c' l='2130' u='r' c='filter_level_for_mb'/>
<offset>64</offset>
<doc f='ffmpeg/libavcodec/vp8.h' l='184'>///&lt; base loop filter level</doc>
