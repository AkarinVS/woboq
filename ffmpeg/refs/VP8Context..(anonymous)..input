<dec f='ffmpeg/libavcodec/vp8.h' l='256' type='const uint8_t *'/>
<use f='ffmpeg/libavcodec/vp8.c' l='841' u='w' c='vp8_decode_frame_header'/>
<offset>0</offset>
