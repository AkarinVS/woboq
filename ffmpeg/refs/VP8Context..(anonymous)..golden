<dec f='ffmpeg/libavcodec/vp8.h' l='275' type='uint8_t'/>
<use f='ffmpeg/libavcodec/vp8.c' l='835' u='w' c='vp8_decode_frame_header'/>
<use f='ffmpeg/libavcodec/vp8.c' l='1299' u='r' c='decode_mb_mode'/>
<offset>48</offset>
